modules/ROOT/pages/_partials/sync/server-deploying-your-app-built-using-the-data-sync-framework-to-a-development-environment.adoc

```diff
@@ -0,0 +1,275 @@
+        - ${NAME}
+        from:
+          kind: ImageStreamTag
+          name: "${NAME}:latest"
+    - type: ConfigChange
+    replicas: 1
+    selector:
+      name: "${NAME}"
+    template:
+      metadata:
+        name: "${NAME}"
+        labels:
+          name: "${NAME}"
+      spec:
+        containers:
+        - name: "${NAME}"
+          image: " "
+          ports:
+          - containerPort:  ${{PORT}}
+          resources:
+            limits:
+              memory: "${MEMORY_LIMIT}"
+          env: []
+parameters:
+- name: NAME
+  displayName: Name
+  description: The name assigned to all of the frontend objects defined in this template.
+  required: true
+  value: data-sync-app
+- name: MEMORY_LIMIT
+  displayName: Memory Limit
+  description: Maximum amount of memory the container can use.
+  required: true
+  value: 512Mi
+- name: SOURCE_REPOSITORY_URL
+  displayName: Git Repository URL
+  description: The URL of the repository with your application source code.
+  required: true
+  value: 'https://github.com/aerogear/voyager-server'
+- name: BUILDER_IMAGE
+  displayName: S2I Builder Image
+  description: The S2I builder image to use when building this application
+  required: true
+  value: nodejs:10
+- name: SOURCE_REPOSITORY_REF
+  displayName: Git Reference
+  description: Set this to a branch name, tag or other ref of your repository if you
+    are not using the default branch.
+- name: CONTEXT_DIR
+  displayName: Context Directory
+  description: Set this to the relative path to your project if it is not in the root
+    of your repository.
+  value: 'examples'
+- name: PORT
+  displayName: Service port
+  description: Port number used by application
+  value: '4000'
+- name: APPLICATION_DOMAIN
+  displayName: Application Hostname
+  description: The exposed hostname that will route to the Node.js service, if left
+    blank a value will be defaulted.
+  value: ''
+- name: GITHUB_WEBHOOK_SECRET
+  displayName: GitHub Webhook Secret
+  description: Github trigger secret.  A difficult to guess string encoded as part
+    of the webhook URL.  Not encrypted.
+  generate: expression
+  from: "[a-zA-Z0-9]{40}"
+- name: GENERIC_WEBHOOK_SECRET
+  displayName: Generic Webhook Secret
+  description: A secret string used to configure the Generic webhook.
+  generate: expression
+  from: "[a-zA-Z0-9]{40}"
+- name: NPM_MIRROR
+  displayName: Custom NPM Mirror URL
+  description: The custom NPM mirror URL
+  value: ''
+----
++
+. Replace the value `https://github.com/aerogear/voyager-server` with the URL for your github repo, and save the file.
++
+. Log in to the environment and browse to a project.
++
+. Click *Add to Project > Import YAML/JSON* to open a screen that allows the user to input raw `yaml` code.
++
+. Copy the contents from the local `yaml` file and paste it into the input box (that was opened in the point above).
++
+. Click *Create* and then *Continue*.
++
+. Review the information and click *Create*.
```

Comment:
when I try this in openshift, after I click on `Create`, I got another prompt asking me choose either 'Process the template' or `Save the template`. I need to choose `Process the template` to continue. I think this step is missing here.

Reply:
Also we should ask user to change the git repo url here.